Cargo.toml

```diff
@@ -34,6 +34,7 @@ esp-idf-sys = { version = "0.32.1", default-features = false, features = ["nativ
 esp-idf-hal = { version = "0.40", default-features = false, features = ["esp-idf-sys"] }
 embassy-sync = { version = "0.1", optional = true }
 embassy-time = { version = "0.1", optional = true, features = ["tick-hz-1_000_000"] }
+once_cell = "1.17.0"
```

Comment:
This introduces a hard dependency on a STD-only crate so that won't fly. See below why we don't need it anyway.